client/app/account/user/credentials/user-credentials.routes.js

```diff
@@ -0,0 +1,10 @@
+angular
+    .module("UserAccount")
+    .config(["$stateProvider", function ($stateProvider) {
```

Comment:
No need to inject $stateProvider before the function, grunt ng-annotate task do it for you :)